src/setup/sync.ts

```diff
@@ -110,21 +115,14 @@ export const sync = (shardus: Shardus, evmCommon: any) => async (): Promise<void
         }
       }
       await sleep(ONE_SECOND * 10)
-
       const when = shardeumGetTime()
-      const existingNetworkAccount = await shardus.getLocalOrRemoteAccount(networkAccount)
-      if (existingNetworkAccount) {
-        /* prettier-ignore */ if (logFlags.important_as_error) shardus.log('NETWORK_ACCOUNT ALREADY EXISTED: ', existingNetworkAccount)
-        await sleep(ONE_SECOND * 5)
```

Comment:
this "wait for 5 seconds" in the case of an existing network account is essentially being removed. I have no idea what the acyuall purpose of it was, maybe it fixed some timing bug, but unless we know why we don't need it anymore, we probably shouldn't remove it